docs/tutorials/proposals/volume-discount-program-proposal.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 11
+title: Enable or replace volume discount program
+hide_title: false
+vega_network: TESTNET
+keywords:
+- proposal
+- governance
+---
+
+import NetworkParameter from '@site/src/components/NetworkParameter';
+import JSONInstructions from './_json-instructions.md';
+import TerminalInstructions from './_terminal-instructions.md';
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+When a volume discount program is enabled, traders can receive discounts on their taker fees. The higher their taker volume over the discount program period, the greater the discount traders can receive.
```

Comment:
> traders can receive discounts on their taker fees

Maybe drop the word taker, parties incur fees on auction exit and discounts are applied here even though their is no concept of a maker/taker.

>over the discount program period

To me, program period implies its your volume in between enactment and closing when its actually your volume over the window specified by the window length.